### PR TITLE
BTRX-825-ORSP: Implement Sample Collection Page

### PR DESCRIPTION
**Pull request by verovicario**
## Addresses
https://broadinstitute.atlassian.net/browse/BTRX-825
## Changes
Add a new component,  Consent Collection Links
## Testing
Go to admin dropdown -> Consent Collection Link
---

- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Get a thumb from Belatrix
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: grails-app/services/org/broadinstitute/orsp/QueryService.groovy [hunk 1]
```diff
@@ -1624,4 +1650,45 @@ class QueryService implements Status {
...
+        String orderField
+        switch (orderColumn) {
+            case 0:
+                orderField = " consent_group_key "
+                break
+            case 1:
+                orderField = " vault_export_date "
+                break
+        }
+        orderField
```
Comment (rushtong) [inline grails-app/services/org/broadinstitute/orsp/QueryService.groovy:1659]: Here and below, could use a default case.

File: grails-app/services/org/broadinstitute/orsp/QueryService.groovy [hunk 2]
```diff
@@ -1624,4 +1650,45 @@ class QueryService implements Status {
...
+                orderField = " sample_collection_id "
+                break
+            case 3:
+                orderField = " status "
+                break
+        }
+        orderField
+    }
+
+    private  getCollectionLinksWhereClause(pagination) {
```
Comment (rushtong) [inline grails-app/services/org/broadinstitute/orsp/QueryService.groovy]: Could use an explicit return type

**Review by rushtong — APPROVED** [review]
Couple minor suggestions inline, but looks good :+1:

**Review by LeoForconesi — APPROVED** [review]

**Review by tadeork — APPROVED** [review]
:+1:

**Pull request merged**